go-controller/pkg/ovn/ovn.go

```diff
@@ -352,14 +352,14 @@ func (oc *DefaultNetworkController) syncNodeGateway(node *kapi.Node, hostSubnets
 			return fmt.Errorf("error cleaning up gateway for node %s: %v", node.Name, err)
 		}
 	} else if hostSubnets != nil {
-		var hostAddrs sets.Set[string]
+		var hostCIDRs sets.Set[string]
 		if config.Gateway.Mode == config.GatewayModeShared {
-			hostAddrs, err = util.ParseNodeHostAddressesDropNetMask(node)
+			hostCIDRs, err = util.ParseNodeHostCIDRsDropNetMask(node)
```

Comment:
this is not a CIDR cause the subnet mask is being dropped, its just a hostAddrs or hostIPs